Update base_import_async.py

OdooMod opened the pull request:
fix error,python csv "delimiter" must be string, not unicode

OdooMod committed:
Update base_import_async.py

lmignon commented:
This one supersedes #23
@waveyeung It seems that you commit is not complete (https://github.com/OCA/connector-interfaces/pull/23/files#diff-ab9d816d577d939a66940f1c6532b2a3R82)

OdooMod commented:
@Imigonon Error msg at travis-ci is "ImportError: No module named connector"?

lmignon commented:
@waveyeung The travis config seems to no more works with the lastest MQT.
Can you remove the line 28 in the file .travis.yml 

```
  - git clone https://github.com/OCA/connector -b ${VERSION} $HOME/connector
```

https://github.com/OCA/connector-interfaces/blob/8.0/.travis.yml#L28 and add an new file named oca_dependencies.txt next to .travis.yml with the following content

```
connector
```

You can find a sample file here https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/oca_dependencies.txt

lmignon commented:
@waveyeung You should include all the changes of others PR #27 #26 #25 into this one.

OdooMod committed:
Update .travis.yml

OdooMod committed:
Create oca_dependencies.txt

lmignon commented on .travis.yml:
```diff
@@ -25,7 +25,7 @@ install:
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
-  - git clone https://github.com/OCA/connector -b ${VERSION} $HOME/connector
+  #- git clone https://github.com/OCA/connector -b ${VERSION} $HOME/connector
```
you can completely remove the line

OdooMod commented:
@lmignon OK. Thanks for help, that's interesting. :)

OdooMod committed:
Update .travis.yml

useless
- git clone https://github.com/OCA/connector -b ${VERSION} $HOME/connector

lmignon commented:
You are welcome. Thanks to you for contributing. 
Nevertheless the changes in base_import_async/models/base_import_async.py seem to be incomplete compared to the PR #23 (https://github.com/OCA/connector-interfaces/pull/23/files#diff-ab9d816d577d939a66940f1c6532b2a3R82)

OdooMod committed:
Update base_import_async.py

lmignon commented:
LGTM :+1: 

For the record you already have one :+1:  from @rvalyi on #23

sbidoul commented:
:+1:
